vtex/address-form
# Add address examples for some countries

#### What is the purpose of this pull request?

Include address examples for some countries when using the geolocation mode.

#### What problem is this solving?

When using the address form in geolocation mode, we use an address from the USA as an input placeholder for any not mapped country, which is not the best option for UX.

We will have new stores in Bulgaria, French, United Kingdom, Germany, Italy, Netherlands, and Poland.

#### How should this be manually tested?

- add this product to cart: https://vtexgame1geo.myvtex.com/checkout/cart/add?sku=312&qty=1&seller=1&redirect=true&sc=2
- use this workspace and navigate until the shipping step: https://vtexgame1geo.myvtex.com/checkout?workspace=garrucho
- change the country to see the new address

#### Screenshots or example usage

![image](https://user-images.githubusercontent.com/7749043/67043255-15d3b200-f100-11e9-8085-8cdc9d338596.png)

#### Types of changes
